scripts/patch-release/patch-release.go

```diff
@@ -55,45 +57,37 @@ func run() error {
 		return fmt.Errorf("open local repo: %w", err)
 	}
 
-	currentBranch, err := repo.CurrentBranch()
-	if err != nil {
-		return fmt.Errorf("get current branch: %w", err)
-	}
-	logrus.Infof("Using current branch: %s", currentBranch)
-
-	newVersion, err := incVersion(version.Version)
-	if err != nil {
-		return fmt.Errorf("increment version: %w", err)
-	}
-
-	logrus.Infof("Using new version: %s", util.SemverToTagString(newVersion))
-
-	if err := updateVersionAndCreatePR(
-		repo, newVersion, currentBranch, org, remote,
-	); err != nil {
-		return fmt.Errorf("update version in local repository: %w", err)
+	currentReleaseBranch := branchPrefix + version.Version
```

Comment:
`currentReleaseBranch` is the base for the PR's we create. This value is now the same for all `releaseMinorVersions`, which is wrong. The base for a PR to update `v1.29.x` should be `release-1.29` and so on.

Reply:
Added the changes as requested.